buddyboss/buddyboss-platform
# Email Invites - Change TinyMCE font to Arial

### All Submissions:


Fixes #630  .

### How to test the changes in this Pull Request:

1. Go To Member profile -> Email Invites Page
2. Check the editor font family, It should be Arial now.

### Proof Screenshots or Video
![Screenshot](https://user-images.githubusercontent.com/53166220/76424943-ec793280-63ce-11ea-9cd8-a29421b25741.png)

### Other information:




### Changelog entry

> Changed Send invites page TinyEMC Editor Font Family to Arial
